File: Application/app/src/main/java/com/example/uniunboxd/DTO/AuthenticationModel.java
package com.example.uniunboxd.DTO;

/**
 * The AuthenticationModel class represents the credentials required for user authentication.
 * It includes an email and a password.
 */
public class AuthenticationModel {
    /**
     * The email of the user.
     */
    public String email;

    /**
     * The password of the user.
     */
    public String password;

    /**
     * Constructs a new AuthenticationModel with the specified email and password.
     * @param email The email of the user.
     * @param password The password of the user.
     */
    public AuthenticationModel(String email, String password) {
        this.email = email;
        this.password = password;
    }
}

File: API/UniUnboxdAPI/Models/DataTransferObjects/AuthenticationModel.cs
namespace UniUnboxdAPI.Models.DataTransferObjects
{
    // AuthenticationModel class for handling authentication data transfer
    public class AuthenticationModel
    {
        // Email property for storing the user's email address
        public required string Email { get; set; }

        // Password property for storing the user's password
        public required string Password { get; set; }
    }
}
